mobile/Ihc/screens/MedicationInventoryScreen.js

```diff
@@ -31,6 +30,33 @@ class MedicationInventoryScreen extends Component<{}> {
 
     this.props.navigator.setOnNavigatorEvent(this.onNavigatorEvent.bind(this));
   }
+  MedicationInventoryForm = t.struct({
```

Comment:
Let's remove all of this if it's defined in the model

Reply:
Wait I misread it... I meant the modal. Either way, it doesn't look like its used here? So let's still get rid of it

Reply:
Changes from Brent's accident